packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -298,7 +301,8 @@ export class HeapDumpService {
   // cloned into a new variable that has a variable name.
   private copyReferenceContainer(
     refContainer: ApexVariableContainer,
-    varName: string
+    varName: string,
+    createVarRef = true
```

Comment:
This was necessary because the variablesRef (which is what tells VS Code that the variable can be expanded) didn't need to be recreated for a variable that's part of the circular reference. The reason cloning was necessary was due to the name changing. For instance local var MyVar of type Foo has field that's a List<Foo> and we add MyVar to its own list. The name of the local variable is MyVar but once added to the list the name is '0' or whatever number in the list it is.